controls/roles/validator-import-api/molecule/teku/create.yml

```diff
@@ -35,7 +35,7 @@
           - "{{ ssh_key_name }}"
         volumes: "{{ item.volumes | default(omit) }}"
         image: "{{ item.image }}"
-        location: "{{ item.location | default(omit) }}"
+        location: "hel1"
```

Comment:
I think this needs to be changed also in all other `validator-import-api` molecule tests, right?